### DISCUSSION TITLE
Unable to sync Shared Folder in 2.5.0

### DISCUSSION THREAD

**Discussion by EpicPilgrim** [abraunegg/onedrive#2812] [Q&A]
I am having trouble getting my shared folder to sync with this new setup. Below is my config:

```
sync_dir = "/home/user/Documents/OneDrive"
enable_logging = "true"
sync_business_shared_items = "true"
```

Below is my sync_list file:

```
Documents - Client - ABC/*
```
_(note that if I use "/Documents - Client - ABC/", it skips the folder entirely. Only seems to work if I drop the leading "/")_

Here is the command I'm running:

```
$ onedrive --confdir=/home/user/.config/onedrive/accounts/OneDrive --monitor -v --resync
```

Here is the relevant bit of the output:

```
Syncing this OneDrive Business Shared Folder: Documents - Client - ABC
Generating a /delta response from the OneDrive API for Drive ID: b!NyrLTpXxxxxu4j3GjGvFtvkB0bld1HZAssYrbxxxxaO6kwGSr_E2QoNyFGU0_TW9
Adding OneDrive folder details for processing
Adding 20 OneDrive items for processing from the OneDrive Shared Folder
Skipping path - excluded by sync_list config: AI Policy
Skipping path - excluded by sync_list config: Applications
Skipping path - excluded by sync_list config: Contracts
Skipping path - excluded by sync_list config: Cybersecurity
Skipping path - excluded by sync_list config: General
Skipping path - excluded by sync_list config: Insurance
Skipping path - excluded by sync_list config: Meetings
Skipping path - excluded by sync_list config: Members
Skipping path - excluded by sync_list config: Pre-Engagement
Skipping path - excluded by sync_list config: Previous ICT Review
Skipping path - excluded by sync_list config: Risk management
Skipping path - excluded by sync_list config: Staff policies
Finished processing self generated /delta JSON response from the OneDrive API
```

What am I missing as to why it is skipping the contents of the OneDrive Shared Folder in sync_list?

**Comment by EpicPilgrim** [comment]
Standby, I think this is likely due to me using the OneDrive Shared Folder instructions and not the Sharepoint Library instructions. I'll try the latter.

**Comment by EpicPilgrim** [comment]
Yep, problem is I was following the [business shared items](https://github.com/abraunegg/onedrive/blob/master/docs/business-shared-items.md) instructions, not the [SharePoint items](https://github.com/abraunegg/onedrive/blob/master/docs/sharepoint-libraries.md) instructions. Got there in the end.

**Comment by abraunegg** [comment]
No problem.

Please ensure you read all the updated documentation carefully. Many things have changed, all docs have been updated.
